de_web_plugin.cpp

```diff
@@ -609,8 +610,16 @@ void DeRestPluginPrivate::apsdeDataIndication(const deCONZ::ApsDataIndication &i
             handleIasZoneClusterIndication(ind, zclFrame);
             break;
 
-        case DE_CLUSTER_ID:
-            handleDEClusterIndication(ind, zclFrame);
+        case VENDOR_CLUSTER_ID:
+        // case DE_CLUSTER_ID:
+            if (zclFrame.manufacturerCode() == VENDOR_PHILIPS)
+            {
+                handlePhilipsClusterIndication(ind, zclFrame);
+            }
+            else // Shouldn't we check for DE manufacturer code?
```

Comment:
Yes could be added (0x1135), or just remove `handleDEClusterIndication()` seems like the function isn't doing much anyway :)

Reply:
Less is more?  Can you remove it on merge, or do you want me to do that?  I'm a bit uncomfortable messing with code I don't understand and cannot test (I don't have any DE devices).